app/scripts/move-popup/dimMoveItemProperties.directive.js

```diff
@@ -186,5 +154,69 @@
         });
       }
     }
+
+    /*
+     * Determins if the item has damange and it's non kinetic.
+     *
+     * @param {Object} item - A item from the destiny server
+     * @return {Boolean}
+     */
+    function itemHasNonKineticDamage(item) {
+      if (!item.dmg || item.dmg === 'kinetic') {
+        return false;
+      }
+      return true;
+    }
+
+    /*
+     * Check for a lockable or elemental property in an item.
+     *
+     * @param {Object} item - A item from the destiny server
+     * @return {Boolean}
```

Comment:
Similar question. Would you guys rather have well named (descriptive) functions without jsdocs? 

I don't typically see jsdocs as a form of educating the reader, but rather properly documenting the code on a system level (maybe that's unnecessary in a side-project like this).